gensim/models/keyedvectors.py

```diff
@@ -617,6 +618,22 @@ def similarity(self, w1, w2):
 
         """
         return dot(matutils.unitvec(self[w1]), matutils.unitvec(self[w2]))
+    
+    def most_similar_to_given(self, w1, word_list):
```

Comment:
Please reformat your docstring according to [google-style](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)